templates/index.html

```diff
@@ -55,6 +55,12 @@
         </section>
     </header>
     <main role="main">
+        <p class="ie-banner">
```

Comment:
Ici ça va l'afficher le temps que la CSS se charge, peut-être qu'on pourrait ajouter un hidden et l'enlever en JS pour IE ? 🤔 /cc @rik

Reply:
Par défaut, les navigateurs ne font pas de rendus tant que les CSS déclarées dans le <head> ne soient téléchargées et interprétées. C'est grâce à ce mécanisme qu'on évite les Flash Of Unstyled Content (FOUC).

Donc non, ça ne devrait pas être un souci.